packages/dom/src/dom/is-text-field.js

```diff
@@ -23,7 +23,9 @@ export default function isTextField( element ) {
 		'number',
 	];
 	return (
-		( nodeName === 'INPUT' && ! nonTextInputs.includes( element.type ) ) ||
+		( nodeName === 'INPUT' &&
+			element.type &&
```

Comment:
`nonTextInputs` is a `string[]` and because `element.type` could be `undefined | string` it doesn't fit in the box of a `string[]`. Checking for it's presence first turns it into a `string` which we can then pass to the `includes` function on the next line.